INSTALLATION.md

```diff
@@ -1,259 +1,250 @@
-# Talawa-API Installation
+# Install/setup node.js
 
-Talawa API is mainly written and built using `node.js` and `typescript`. We are providing some common approaches to set it up on your system.
+Best way to install and manage node.js is making use of node managers. Two most popular node managers right now are [fnm](https://github.com/Schniz/fnm) and [nvm](https://github.com/nvm-sh/nvm). We'd recommend fnm because it's much faster than nvm. Install whichever one you want and follow their guide to set up node.js on your system.
```

Comment:
Why is this section here if we don't appear to refer to using either `nvm` or `fnm` commands later on?

Reply:
For installing node.js?

Reply:
I missed the context and the heading. OK